scripts/run-fork-chain.sh

```diff
@@ -66,11 +66,12 @@ then
     echo "Cloning repository..."
     git clone https://github.com/purestake/moonbeam
     cd $ROOT_FOLDER/moonbeam
+    git checkout $GIT_TAG
```

Comment:
Git_tag should be checked out regardless if you've set `SKIP_COMPILATION` on or off, as otherwise you'd be always forced to use `master` for other files such as `state-modifier.ts`